playground/mocks/data/idp/idx/authenticator-verification-phone-2.json

```diff
@@ -0,0 +1,269 @@
+                      }
+                    ]
+                  }
+                }
+              },
+              {
+                "label": "Phone Number",
+                "value": {
+                  "form": {
+                    "value": [
+                      {
+                        "name": "id",
+                        "required": true,
+                        "value": "aut1boqHwwWipILIM0g4",
+                        "mutable": false
+                      },
+                      {
+                        "name": "methodType",
+                        "required": true,
+                        "value": "voice",
+                        "mutable": false
+                      },
+                      {
+                        "name": "phoneNumber",
+                        "required": true
+                      }
+                    ]
+                  }
+                }
+              }
+            ]
+          },
+          {
+            "name": "stateHandle",
+            "required": true,
+            "value": "02WTSGqlHUPjoYvorz8T48txBIPe3VUisrQOY4g5N8",
+            "visible": false,
+            "mutable": false
+          }
+        ]
+      }
+    ]
+  },
+  "authenticatorEnrollments": {
+    "type": "array",
+    "value": [
+      {
+        "label": "Okta Password",
+        "value": {
+          "type": "password",
+          "authenticatorId": "autwa6eD9o02iBbtv0g1",
+          "id": "aidwboITrg4b4yAYd0g3"
+        }
+      },
+      {
+        "label": "Security Key or Biometric Authenticator (FIDO2)",
+        "value": {
+          "type": "webauthn",
+          "authenticatorId": "autwa6eD9o02iBbtv0g2",
+          "id": "aidtheidkwh282hv8g3"
+        }
+      },
+      {
+        "label": "Okta Email",
+        "value": {
+          "type": "email",
+          "id": "aidtm56L8gXXHI1SD0g3",
+          "authenticatorId": "autwa6eD9o02iBbtv0g3",
+          "methods": [
+            {
+              "methodType": "email"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  "currentAuthenticatorEnrollment": {
+    "type": "object",
+    "label":"Okta Phone",
+    "value": {
+      "type": "phone",
+      "id": "aidtm56L8gXXHI1SD0g3",
+      "methods": [
+        {
+          "methodType": "voice"
+        },
+        {
+          "methodType": "sms"
+        }
+      ],
+      "profile": {
+        "phone": "+1 XXX-XXX-2342"
+      },
+      "resend": {
+        "rel": [
+          "create-form"
+        ],
+        "name": "resend",
+        "href": "http://localhost:3000/idp/idx/challenge/resend",
+        "method": "POST",
+        "accepts": "application/vnd.okta.v1+json",
+        "value": [
+          {
+            "name": "stateHandle",
+            "required": true,
+            "value": "02WTSGqlHUPjoYvorz8T48txBIPe3VUisrQOY4g5N8",
+            "visible": false,
+            "mutable": false
+          }
+        ]
+      },
+      "poll": {
```

Comment:
- don't think we support polling at phon verification step. this is probably copy from email and let's remove it.
- same to other mock files

Reply:
Do we still need polling mechanism then? I have code for polling, not sure if it makes sense for phone verification now..

Reply:
no. we shall remove polling from code as well.